### PR TITLE
Add wb32-dfu-updater_cli util

### PR DESCRIPTION
**Pull request by wb-Joy**
Add wb32-dfu-updater_cli util.

### CODE REVIEW

File: mingw-w64-wb32-dfu-updater_cli/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+pkgver=0.0.3
+pkgrel=1
+pkgdesc='Device firmware upgrade utilities (mingw-w64)'
+arch=('any')
+mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64' 'clang32')
+license=('Apache2.0')
+groups=("${MINGW_PACKAGE_PREFIX}-eda")
+url='https://github.com/WestberryTech/wb32-dfu-updater/'
+depends=("${MINGW_PACKAGE_PREFIX}-libusb")
+makedepends=("${MINGW_PACKAGE_PREFIX}-cmake")
```
Comment (MehdiChinoune) [inline mingw-w64-wb32-dfu-updater_cli/PKGBUILD:15]:
```suggestion
makedepends=("${MINGW_PACKAGE_PREFIX}-cmake"
             "${MINGW_PACKAGE_PREFIX}-ninja")
```

File: mingw-w64-wb32-dfu-updater_cli/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+source=("${_realname}-${pkgver}.tar.gz::https://github.com/WestberryTech/wb32-dfu-updater/archive/refs/tags/v${pkgver}.tar.gz")
+sha256sums=('63fd8c77bfc4345f9ed6cbde0cff431a3afcfdc57808ad76c413a43c3f56f942')
+
+prepare() {
+    cd "${srcdir}/${_realname}-${pkgver}"
+}
+
+build() {
+    [[ -d "${srcdir}/build-${MINGW_CHOST}" ]] && rm -rf "${srcdir}/build-${MINGW_CHOST}"
+    mkdir -p "${srcdir}/build-${MINGW_CHOST}" && cd "${srcdir}/build-${MINGW_CHOST}"
```
Comment (MehdiChinoune) [inline mingw-w64-wb32-dfu-updater_cli/PKGBUILD:25]:
```suggestion
    [[ -d "${srcdir}/build-${MSYSTEM}" ]] && rm -rf "${srcdir}/build-${MSYSTEM}"
    mkdir -p "${srcdir}/build-${MSYSTEM}" && cd "${srcdir}/build-${MSYSTEM}"
```

File: mingw-w64-wb32-dfu-updater_cli/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+    MSYS2_ARG_CONV_EXCL="-DCMAKE_INSTALL_PREFIX=" \
+    ${MINGW_PREFIX}/bin/cmake.exe \
+      -G"MinGW Makefiles" \
```
Comment (MehdiChinoune) [inline mingw-w64-wb32-dfu-updater_cli/PKGBUILD:29]:
```suggestion
      -G"Ninja" \
```

File: mingw-w64-wb32-dfu-updater_cli/PKGBUILD [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+      -DCMAKE_INSTALL_PREFIX=${MINGW_PREFIX} \
+      -DCMAKE_BUILD_TYPE=Release \
+      ../${_realname}-${pkgver}
+
+    ${MINGW_PREFIX}/bin/cmake.exe --build ./
+}
+
+package() {
+    cd "${srcdir}/build-${MINGW_CHOST}"
```
Comment (MehdiChinoune) [inline mingw-w64-wb32-dfu-updater_cli/PKGBUILD:38]:
```suggestion
    cd "${srcdir}/build-${MSYSTEM}"
```

File: mingw-w64-wb32-dfu-updater_cli/PKGBUILD [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+    DESTDIR="${pkgdir}" ${MINGW_PREFIX}/bin/cmake.exe --build ./ --target install
```
Comment (MehdiChinoune) [inline mingw-w64-wb32-dfu-updater_cli/PKGBUILD:40]:
```suggestion
    DESTDIR="${pkgdir}" ${MINGW_PREFIX}/bin/cmake.exe --install ./
```

**Comment by wb-Joy** [comment]
Is there anything else that needs to be done before he can be merged?

**Comment by MehdiChinoune** [comment]
> Is there anything else that needs to be done before he can be merged?

You must be patient, PRs sometimes took weeks to get merged in this repo.

**Comment by wb-Joy** [comment]
> > Is there anything else that needs to be done before he can be merged?
> 
> You must be patient, PRs sometimes took weeks to get merged in this repo.

All right.

**Comment by lazka** [comment]
Upstream is gone (??)

**Comment by wb-Joy** [comment]
> Upstream is gone (??)

I'll resubmit it.

**Comment by wb-Joy** [comment]
New PR #10420.

**Comment by wb-Joy** [comment]
Has been restored.

**Pull request merged**